environs/manual/common.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package manual
+
+import (
+	"net"
+
+	"github.com/juju/errors"
+
+	"github.com/juju/juju/apiserver/params"
+)
+
+var netLookupHost = net.LookupHost
+
+var (
+	// ErrProvisioned is returned by ProvisionMachine if the target
+	// machine has an existing machine agent.
+	ErrProvisioned = errors.New("machine is already provisioned")
```

Comment:
Sorry I missed this before, this should also go in the other file, as it is also part of the API.